docs/application/native/guides/internationalization/i18n.md

```diff
@@ -773,6 +777,101 @@ The `current()` and `next()` functions only check the current index against the
 
 The UCharIter functions return code unit values in the range of 0 - 0xffff. Before any functions operating on strings are called, the string must be set with the `i18n_uchar_iter_set_string()`, `i18n_uchar_iter_set_UTF16BE()`, or `i18n_uchar_iter_set_UTF8()`function.
 
+
+<a name="plural_format"></a>
+## Format Plural Messages with PluralFormat
+
+The PluralFormat API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__PLURAL__FORMAT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__PLURAL__FORMAT__MODULE.html) applications) allows you to [format defined plural words](#plural_format_examples). Alongside with PluralRules module, you can define rules to which format would apply.
+
+PluralFormat supports creation of locale-dependent messages with plural inflection. To enable plural formatting,
+you have to specify each message for every plural case that will be used, based on the desired language.
+Then, the PluralFormat module will select appropriate text based on the provided number.
+
+### Issues of Plural Forms in Internationalized Messages
+
+Many languages have different approach to create plural nouns. The locale-dependent message is workable when the framework
+enables to manage plural forms of all language correctly. For example, in Polish, there is different plural case for nouns
+counted as few (numbers as 2, 3 or 4) and in case with many objects (more than 5).
+
+PluralFormat deals with this by diving the problem into two parts:
```

Comment:
Please change "diving" to "dividing" and also please maintain consistency in writing Plural Format as either one word or separate word in this file.